modal/_container_entrypoint.py

```diff
@@ -490,10 +490,12 @@ def main(container_args: api_pb2.ContainerArguments, client: Client):
                 client,
             )
 
+        # Get ids and metadata for objects (primarily functions and classes) on the app
+        container_app: RunningApp = container_io_manager.get_app_objects()
```

Comment:
Any functional consequences of moving this out of the if block below?

Reply:
I noticed we're actually referring to it in the higher-level scope, in the next block. We basically had code like this

```python
if a:
    x = X()

if b:
    x.f()
```

I _think_ that the 2nd block is never entered unless the 1st block is also entered, so I don't think there's a practical difference. I'm surprised mypy or ruff doesn't catch issues like this actually.